draft-ietf-rats-corim.md

```diff
@@ -1254,11 +1254,28 @@ applies to all measurements in the triple, including those in `measurement-value
 # CoBOM {#sec-cobom}
 
 A Concise Bill of Material (CoBOM) object represents the signal for the
-Verifier to activate the listed tags. Data contained in a tag MUST NOT be used
+Verifier to activate the listed tags. Data contained in a tag SHOULD NOT be used
 for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's Appraisal Policy for Evidence, which is often driven by the complexity and nature of
+the use case.
+
+For example, in cases where only limited CoRIMs are produced by a single authority, a Verifier Owner may have
+a policy that CoBOM's are not required. In this case, signature chain over each CoRIM is sufficient authorization,
+and may not require additional authorization via CoBOM. Any tags in a CoRIM received by a Verifier
+are activated immediately and treated valid for appraisal
+
+However, a more common case is the one where Verifier receives multiple CoRIMs containing
+Reference Values and Endorsements from multiple Reference Value providers and Endorsers
+respectively. In such cases, an integrator or a supplier authority may be designated to
+issue a single CoBOM to activate all the CoRIM tags.
+
+In a more complex case, there may be multiple authorities that issue CoBOM's at different points in time.
+An Appraisal Policy for Evidence may dictate how multiple CoBOM's are to be processed within the Verifier.
```

Comment:
```suggestion
An Appraisal Policy for Evidence may dictate how multiple CoBOM are to be processed within the Verifier.
```

Reply:
The examples from lines 1266 to 1277 are too "value judgement" focused as evidenced by terms like "common" and "more complex".

Reply:
> The examples from lines 1266 to 1277 are too "value judgement" focused as evidenced by terms like "common" and "more complex".

Yes, this is required as to provide some clarity as to why there are cases, when no CoBOM, 1 CoBOM and N CoBOM are relevant.

Else, it is hard for a reader to grasp the real requirement or applicability of the CoBOM.

Reply:
> Yes, this is required as to provide some clarity as to why there are cases, when no CoBOM, 1 CoBOM and N CoBOM are relevant.
> 
> Else, it is hard for a reader to grasp the real requirement or applicability of the CoBOM.

The explanation doesn't describe the 0,1,** property, rather it tries to describe it in exemplary terms. Maybe it should describe the considerations for 0,1,** and the algorithm for processing them. Exemplary text can add concreteness in terms of a use case. But the description of the use case doesn't need to imply what is common or simple etc.

Reply:
> > Yes, this is required as to provide some clarity as to why there are cases, when no CoBOM, 1 CoBOM and N CoBOM are relevant.
> > Else, it is hard for a reader to grasp the real requirement or applicability of the CoBOM.
> 
> The explanation doesn't describe the 0,1,** property, rather it tries to describe it in exemplary terms. Maybe it should describe the considerations for 0,1,** and the algorithm for processing them. Exemplary text can add concreteness in terms of a use case. But the description of the use case doesn't need to imply what is common or simple etc. I imagine something like the following:
> 
> * When no CoBOMs exist, the Verifier does ...
> * When 1 CoBOM exists, the Verifier does ...
> * When multiple CoBOMs exist, the Verifier does ...

I have done shorter version now and focused more on what Verifier does, when no CoBOM, one CoBOM and multiple CoBOM is the case!